app/assets/stylesheets/components/beta_tag/_beta_tag.scss

```diff
@@ -0,0 +1,11 @@
+.beta-tag {
+  @include inline-block();
+  @include body(18, 24);
+  font-weight: 700;
+  position: relative;
+  bottom: -$baseline-unit*5;
```

Comment:
You could zero the margin-top on the `h1` and then just have a margin-bottom on the beta tag to achieve the same thing right? 

But as it's a temporary element this works. In general negative bottom margins can mean headaches later though. Also shall we both agree limit use of the phrase `negative bottom margins`?